source/tutorials/feature-classifier.rst

```diff
@@ -16,7 +18,7 @@ We will download and create several files, so first create a working directory.
 Obtaining and importing reference data sets
 -------------------------------------------
 
-Two elements are required for training the classifier: the reference sequences and the corresponding taxonomic classifications. To reduce computation time for this tutorial we will use the relatively small `Greengenes`_ 13_8 85% OTU data set.
+Two elements are required for training the classifier: the reference sequences and the corresponding taxonomic classifications. To reduce computation time for this tutorial we will use the relatively small `Greengenes`_ 13_8 85% OTU data set. **Do not use the 85% OTUs for training classifiers to be used on real data**. We recommend using more information-rich sequences, e.g., reference sequences clustered at 99% sequence similarity, for classification of real data. See the QIIME 2 :doc:`data resources page <../data-resources>` for links to complete QIIME-compatible reference datasets.
```

Comment:
"Do not use the 85% OTUs for training classifiers to be used on real data"
->
"Do not use the 85% OTU data set used in this tutorial on real data."

Or something along those line...